### ISSUE TITLE
Broken in next 12.1.6

### ISSUE THREAD

**Issue by EvHaus** [remorses/next-superjson#5]
After upgrading to next 12.1.6 the plugin now fails with:

```
ValidationError: Invalid configuration object. Webpack has been initialized using a configuration object that does not match the API schema.
 - configuration[0].module.rules[6] should be one of these:
   ["..." | object { assert?, compiler?, dependency?, descriptionData?, enforce?, exclude?, generator?, include?, issuer?, issuerLayer?, layer?, loader?, mimetype?, oneOf?, options?, parser?, realResource?, resolve?, resource?, resourceFragment?, resourceQuery?, rules?, scheme?, sideEffects?, test?, type?, use? }, ...]
   -> A rule.
   Details:
    * configuration[0].module.rules[6].include[0] has an unknown property 'pages'. These properties are valid:
      object { and?, not?, or? }
      -> Logic operators used in a condition matcher.
    * configuration[0].module.rules[6].include[0] has an unknown property 'root'. These properties are valid:
      object { and?, not?, or? }
      -> Logic operators used in a condition matcher.
    * configuration[0].module.rules[6].include[0] has an unknown property 'pages'. These properties are valid:
      object { and?, not?, or? }
      -> Logic operators used in a condition matcher.
    * configuration[0].module.rules[6].include[0] has an unknown property 'root'. These properties are valid:
      object { and?, not?, or? }
      -> Logic operators used in a condition matcher.
 - configuration[1].module.rules[5] should be one of these:
   ["..." | object { assert?, compiler?, dependency?, descriptionData?, enforce?, exclude?, generator?, include?, issuer?, issuerLayer?, layer?, loader?, mimetype?, oneOf?, options?, parser?, realResource?, resolve?, resource?, resourceFragment?, resourceQuery?, rules?, scheme?, sideEffects?, test?, type?, use? }, ...]
   -> A rule.
   Details:
    * configuration[1].module.rules[5].include[0] has an unknown property 'pages'. These properties are valid:
      object { and?, not?, or? }
      -> Logic operators used in a condition matcher.
    * configuration[1].module.rules[5].include[0] has an unknown property 'root'. These properties are valid:
      object { and?, not?, or? }
      -> Logic operators used in a condition matcher.
    * configuration[1].module.rules[5].include[0] has an unknown property 'pages'. These properties are valid:
      object { and?, not?, or? }
      -> Logic operators used in a condition matcher.
    * configuration[1].module.rules[5].include[0] has an unknown property 'root'. These properties are valid:
      object { and?, not?, or? }
      -> Logic operators used in a condition matcher.
 - configuration[2].module.rules[6] should be one of these:
   ["..." | object { assert?, compiler?, dependency?, descriptionData?, enforce?, exclude?, generator?, include?, issuer?, issuerLayer?, layer?, loader?, mimetype?, oneOf?, options?, parser?, realResource?, resolve?, resource?, resourceFragment?, resourceQuery?, rules?, scheme?, sideEffects?, test?, type?, use? }, ...]
   -> A rule.
   Details:
    * configuration[2].module.rules[6].include[0] has an unknown property 'pages'. These properties are valid:
      object { and?, not?, or? }
      -> Logic operators used in a condition matcher.
    * configuration[2].module.rules[6].include[0] has an unknown property 'root'. These properties are valid:
      object { and?, not?, or? }
      -> Logic operators used in a condition matcher.
    * configuration[2].module.rules[6].include[0] has an unknown property 'pages'. These properties are valid:
      object { and?, not?, or? }
      -> Logic operators used in a condition matcher.
    * configuration[2].module.rules[6].include[0] has an unknown property 'root'. These properties are valid:
      object { and?, not?, or? }
      -> Logic operators used in a condition matcher.
    at validate (/home/myproject/node_modules/next/dist/compiled/schema-utils3/index.js:1:150822)
    at validateSchema (/home/myproject/node_modules/next/dist/compiled/webpack/bundle5.js:131217:2)
    at create (/home/myproject/node_modules/next/dist/compiled/webpack/bundle5.js:134714:24)
    at webpack (/home/myproject/node_modules/next/dist/compiled/webpack/bundle5.js:134761:32)
    at Object.f [as webpack] (/home/myproject/node_modules/next/dist/compiled/webpack/bundle5.js:89273:16)
    at HotReloader.start (/home/myproject/node_modules/next/dist/server/dev/hot-reloader.js:473:45)
    at async DevServer.prepare (/home/myproject/node_modules/next/dist/server/dev/next-dev-server.js:298:9)
    at async /home/myproject/node_modules/next/dist/cli/next-dev.js:127:9 {
  errors: [
    {
      keyword: 'oneOf',
      dataPath: '[0].module.rules',
      schemaPath: '#/properties/rules/oneOf',
      params: [Object],
      message: 'should match exactly one schema in oneOf',
      schema: [Array],
      parentSchema: [Object],
      data: [Array],
      children: [Array]
    },
    {
      keyword: 'oneOf',
      dataPath: '[1].module.rules',
      schemaPath: '#/properties/rules/oneOf',
      params: [Object],
      message: 'should match exactly one schema in oneOf',
      schema: [Array],
      parentSchema: [Object],
      data: [Array],
      children: [Array]
    },
    {
      keyword: 'oneOf',
      dataPath: '[2].module.rules',
      schemaPath: '#/properties/rules/oneOf',
      params: [Object],
      message: 'should match exactly one schema in oneOf',
      schema: [Array],
      parentSchema: [Object],
      data: [Array],
      children: [Array]
    }
  ]
```

**Comment by remorses** [comment]
FIxed in 0.0.3

**Comment by EvHaus** [comment]
Confirmed! Thanks for the quick fix.
